### ISSUE TITLE
existingClaim does not refer PVC in victoria-metrics-single chart

### ISSUE THREAD

**Issue by challapradyumna** [VictoriaMetrics/helm-charts#189]
existingClaim does not create a PVC but does not refer to the provided value in sts persistentVolumeClaim. 

https://github.com/VictoriaMetrics/helm-charts/blob/master/charts/victoria-metrics-single/templates/server-statefulset.yaml

**Comment by f41gh7** [comment]
If i understand correctly, it is possible to fix with extraVolumes. Something like:

```yaml
server:
   persistentVolume:
       enabled: false
server:
  extraVolumes:
  - name: server-volume
    persistentVolumeClaim:
      name: exist-claim
```

**Comment by challapradyumna** [comment]
Solved it by naming the PVC whatever the chart expects it to be. 
Isn't the `exisistingClaim` value supposed to do it automatically. If not docs have to be updated with that.

**Comment by AndrewChubatiuk** [comment]
this issue was finally fixed in release [0.11.0](https://github.com/VictoriaMetrics/helm-charts/releases/tag/victoria-metrics-single-0.11.0)
